schutzbot/prepare-rhel-internal.sh

```diff
@@ -8,7 +8,7 @@ function greenprint {
 
 ARCH=$(uname -m)
 
-source ../tools/define-compose-url.sh
```

Comment:
Did this work for you in Jenkins @atodorov ? I had to change for Gitlab CI, so in case this was correct I need to add some if condition.

Reply:
It works in Jenkins b/c we execute from inside the `schutzbot` directory. Guess pwd is different for GitLab.

Reply:
Alright, I solved this in the first way I could come up with just to keep Jenkins compatibility until we're certain we can use Gitlab CI for nightly testing. I guess I'll add a comment with this information too.